docs/source/development/style-guide/review-standards.rst

```diff
@@ -0,0 +1,65 @@
+.. _pull-request-review-standards:
+
+Pull Request and Review Standards
+---------------------------------
+
+Before any code is merged into our code base, it will need to be put up for pull request (PR) and reviewed. The pull request should be created following
+the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>`.
+
+Before opening pull request
+===========================
+
+Before you create the pull request, you should go through the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>` to ensure
+the proposed changes are required and up to our standards.
+
+The PR should be as small as possible. However, the code included in the PR should be complete. It should complete an entire feature, but this doesn't necessarily mean it completes an entire issue.
+However, all code merged into the ``dev`` branch should be up to our standards and work sufficiently.
+
+If you want to work on the pull request or are not yet finished with the code, please indicate this by marking the pull request as a
+`draft or WIP <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests>`_ PR.
+Anyone looking at the PR will be able to quickly see it is not yet ready for review.
+
+Finally, if you are addressing an existing issue, make sure that issue is linked in your PR. If there is not an existing issue, then you should either create an issue or address WHY you are opening the PR specifically.
+If your PR addresses a `Level-5 requirement <https://github.com/orgs/IMAP-Science-Operations-Center/projects/2/views/2>`_, there **must** be a corresponding issue linked.
+
+During review
+=============
+
+As a reviewer, please follow these rules of thumb:
+
+#. Comments should be clear in addressing why you want to see the change
+#. Comments should be polite, but straightforward and candid
+#. If you leave a review, continue to follow up on replies to your questions or comments and review the changes you requested
+#. It is polite, but not required, to provide examples for suggestions (particularly for things like name changes)
+#. If you require a change to be addressed, add a "request changes" comment. If you make one of these comments, it means you are blocking the code review from merging until that change is addressed.
+#. If you make a "request changes" comment, you must create a follow up review where you change that to an approving review. Please do this in a timely matter so you do not block the PR for longer than necessary
```

Comment:
```suggestion
#. If you make a "request changes" comment, you must follow up and dismiss your "request changes" review once the author has addressed that aspect of the code. Please do this in a timely manner so you do not block the PR for longer than necessary
```

I am hesitant to say that I have to change it to an approving review. I have sometimes wanted to just dismiss my request changes, but still don't feel confident in fully approving a code change. e.g. if I was blocking a small portion like a name change, but I don't have enough expertise in the broader larger code changes to speak for an approval.